packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx

```diff
@@ -198,7 +198,8 @@ const PullRequestListPage: FC<PullRequestPageProps> = ({
           <>
             <PRListFilterHandler
               ref={filtersRef}
-              onFilterSelectionChange={onFilterSelectionChange}
+              // TODO: fix type issue
+              {...{ onFilterSelectionChange: onFilterSelectionChange }}
```

Comment:
I'm not seeing any type issues here 🤔 
```suggestion
              onFilterSelectionChange={onFilterSelectionChange}
```

Reply:
The issue has gone after rebase👌